frontend/packages/policy-editor/src/components/ExpandablePolicyCard/ExpandablePolicyCard.tsx

```diff
@@ -199,8 +199,14 @@ export const ExpandablePolicyCard = ({
   /**
    * Displays the actions
    */
-  const displayActions = policyRule.actions.map((a, i) => {
-    return <ActionAndSubjectListItem key={i} title={a} onRemove={() => handleRemoveAction(i, a)} />;
+  const displayActions = policyRule.actions.map((actionId, i) => {
+    return (
+      <ActionAndSubjectListItem
+        key={i}
+        title={t(`policy_editor.action_${actionId}`)}
```

Comment:
Should we default to using the `actionId` if the translation is not found in the language files? It might be worth considering extracting this functionality into a separate function. 😊 

```suggestion
        title={getTranslationByActionId(actionId)}
```
```typescript
getTranslationByActionId(actionId: string): string | null => {
  return t(`policy_editor.action_${actionId}`) || actionId;
}
```